pkg/machine/pull.go

```diff
@@ -177,21 +177,44 @@ func Decompress(localPath, uncompressedPath string) error {
 // Will error out if file without .xz already exists
 // Maybe extracting then renameing is a good idea here..
 // depends on xz: not pre-installed on mac, so it becomes a brew dependency
-func decompressXZ(src string, output io.Writer) error {
-	cmd := exec.Command("xzcat", "-k", src)
-	//cmd := exec.Command("xz", "-d", "-k", "-v", src)
-	stdOut, err := cmd.StdoutPipe()
-	if err != nil {
-		return err
+func decompressXZ(src string, output io.WriteCloser) error {
+	var read io.Reader
+	var cmd *exec.Cmd
+	// Prefer xz utils for fastest performance, fallback to go xi2 impl
+	if _, err := exec.LookPath("xzcat"); err == nil {
+		cmd = exec.Command("xzcat", "-k", src)
+		//cmd := exec.Command("xz", "-d", "-k", "-v", src)
```

Comment:
Please don't add commented out code. This looks like the comment does not match the code.

Reply:
Sorry not sure why I didn't delete that when I updated the code